docs/Developers/Review.rst

```diff
@@ -91,6 +91,8 @@ request but should ideally pass for every commit.
 For the Reviewers
 -----------------
 
+-  Read and try to understand the code. If something looks ineffective or
```

Comment:
we have docs for writing tests. Maybe link to them with a "make sure tests follow <docs>" instead of having that info in two places?